scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -85,45 +84,78 @@ public static BuildTargetResult Compile(BuildTargetContext c)
         [Target]
         public static BuildTargetResult GenerateStubHostPackages(BuildTargetContext c)
         {
-            string currentRid = GetRuntimeId();
             var buildVersion = c.BuildContext.Get<BuildVersion>("BuildVersion");
-            PrepareDummyRuntimeNuGetPackage(
-                            DotNetCli.Stage0,
-                            buildVersion.HostNuGetPackageVersion,
-                            Dirs.CorehostDummyPackages);
-            foreach (var hostPackageId in HostPackages)
+            string currentRid = GetRuntimeId();
+            PrepareDummyRuntimeNuGetPackage(DotNetCli.Stage0);
+            foreach (var hostPackage in buildVersion.LatestHostPackages)
             {
-                foreach (var rid in HostPackageSupportedRids)
+                foreach (var rid in HostPackageSupportedRids.Keys)
                 {
                     if (!rid.Equals(currentRid))
```

Comment:
Would it make sense to check the value here instead of the keys?

For example if we're on osx10.10, do we still need to generate a osx10.10 package for osx10.11?